ui/addon/src/main/java/org/jboss/windup/ui/WindupUpdateRulesetCommand.java

```diff
@@ -62,6 +62,11 @@ public Result execute(UIExecutionContext context) throws Exception
         Coordinate latestCoordinate;
         do{
             i++;
+            i++;
+            if(resolveVersions.size() <= i) {
+                //if we haven't found final version
+                return Results.fail("Have not found any online Final version of the ruleset.");
```

Comment:
This message should probably be changed to:

"Could not check for new rulesets because the online repository did not return version information - please try again later. If this error continues to occur, contact the Windup project team at windup-users@lists.jboss.org."